docs/jax-101/01-jax-basics.md

```diff
@@ -254,13 +254,13 @@ in_place_modify(jnp.array(x))  # Raises error when we cast input to jnp.ndarray
 
 +++ {"id": "RGqVfYSpc49s"}
 
-Helpfully, the error points us to JAX's side-effect-free way of doing the same thing via the [`jax.ops.index_*`](https://jax.readthedocs.io/en/latest/jax.ops.html#indexed-update-operators) ops. They are analogous to in-place modification by index, but create a new array with the corresponding modifications made:
+Helpfully, the error points us to JAX's side-effect-free way of doing the same thing via the [`jax.ops.*`](https://jax.readthedocs.io/en/latest/jax.ops.html#indexed-update-operators) index update operators (be careful [`jax.ops.index_*`](https://jax.readthedocs.io/en/latest/jax.ops.html#indexed-update-functions-deprecated) functions are deprecated). They are analogous to in-place modification by index, but create a new array with the corresponding modifications made:
```

Comment:
It might be more clear to link directly to the documentation for the `jnp.ndarray.at` property: https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.ndarray.at.html

Reply:
I used the same link that is provided in the error message, I think it's better to have it the same?

Reply:
Oh, I missed that. The error message should probably be updated as well – are you willing to do that as part of this PR?

Reply:
OK, I'll do it

Reply:
I did it.